src/main/java/com/vinylteam/vinyl/entity/Vinyl.java

```diff
@@ -4,13 +4,29 @@
 
 public class Vinyl {
 
+    private long vinylId;
+    private int shopId;
+    private long uniqueVinylId;
```

Comment:
Should we add this row into the hashCode() and toString() methods?

Reply:
toString is not present in the Vinyl class - I will add it. And I add these three fields to hashCode and equals